pkg/spoke/spokeagent.go

```diff
@@ -172,6 +172,7 @@ func (o *SpokeAgentOptions) RunSpokeAgent(ctx context.Context, controllerContext
 			return err
 		}
 
+		controllerName := fmt.Sprintf("BootstrapClientCertController@cluster:%s", o.ClusterName)
```

Comment:
I  think this line should be out of if? What if the agent is bootstrapped and restarted?